auto_tune_vllm/cli/main.py

```
@@ -0,0 +1,877 @@
+@app.command("validate")
+def validate_command(
+    config: str = typer.Option(..., "--config", "-c", help="Configuration file to validate"),
+):
+    """Validate study configuration file."""
+    console.print(f"[blue]Validating configuration: {config}[/blue]")
+    
+    try:
+        config_path = Path(config)
+        if not config_path.exists():
+            console.print(f"[bold red]Error: File not found: {config}[/bold red]")
+            raise typer.Exit(1)
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)